app/src/main/java/com/jakdor/apapp/ui/login/LoginViewModel.kt

```diff
@@ -11,8 +12,21 @@ class LoginViewModel
                     rxSchedulersFacade: RxSchedulersFacade):
     BaseViewModel(application, rxSchedulersFacade){
 
-    fun isEditTextEmpty (etText : EditText) : Boolean {
-        return etText.text.toString().trim().isEmpty()
+
+    val loginPossibility = MutableLiveData<Boolean>().apply { value = false }
+
+    var isLoginFilled: Boolean = false
+    var isPasswordFilled: Boolean = false
+
+    fun checkLoginFilled (etText : EditText)  {
```

Comment:
Jeszcze się przypierdolę, by do ViewModeli zwracać tylko bazowe typy zmiennych typu stringi np., bo teraz mamy viewModel zależny od implementacji widoku, tak poza tym to zrozumiałeś o co mi chodziło.